views/templates/home.html.twig

```diff
@@ -3,7 +3,7 @@
 
 {% block body %}
 	<div class="home__form mt-5 d-flex justify-content-center">
-		<form action="/custom_cms/home?insert" class = "p-4" method = "POST">
+		<form action="/cms/home?insert" class = "p-4" method = "POST">
```

Comment:
can you remove this `/cms` part ? since our application doesn't need the baseUrl ?